flocker/node/agents/blockdevice.py

```diff
@@ -90,6 +90,8 @@ class DiscoveredDataset(PClass):
         the node where the dataset is attached.
     :ivar FilePath mount_point: The absolute path to the location on the node
         where the dataset will be mounted.
+    :ivar FilePath share_path: The absolute path to the location on the node
+        where the dataset is available for use by an external client.
```

Comment:
I think it would be helpful to give an example, or perhaps to say something about how `share_path` relates to `mount_point`.